qutip/solver/brmesolve.py

```diff
@@ -285,6 +285,8 @@ def __init__(self, H, a_ops, c_ops=None, sec_cutoff=0.1, *, options=None):
         self._integrator = self._get_integrator()
         self._state_metadata = {}
         self.stats = self._initialize_stats()
+        self.sys_dims = self.rhs.dims[0]
```

Comment:
I will suggest making this a property if it always depends rhs.

Reply:
`HEOMSolver` is the exception. It has the `_sys_dims` parameter for the physical dimensions, but rhs is expanded to include the bath.
But a property make sense, I don't see any reason to overwrite the value.